src/v/features/feature_table.cc

```diff
@@ -724,8 +727,19 @@ void feature_table::set_builtin_trial_license(
 
 void feature_table::revoke_license() { _license = std::nullopt; }
 
-const std::optional<security::license>& feature_table::get_license() const {
-    return _license ? _license : _builtin_trial_license;
+const std::optional<security::license>&
+feature_table::get_license(bool use_bootstrap_license) const {
+    if (_license) {
+        return _license;
+    } else if (_builtin_trial_license) {
+        return _builtin_trial_license;
```

Comment:
Is it possible to not set this? `revoke_license` doesn't unset this.

My goal is to test behaviour on license expiry.

Reply:
Ahh: `__REDPANDA_DISABLE_BUILTIN_TRIAL_LICENSE`

Reply:
Yeah `__REDPANDA_DISABLE_BUILTIN_TRIAL_LICENSE ` can be used for ducktape tests, `revoke_license` for unit/fixture tests.